system/CodeIgniter.php

```diff
@@ -101,7 +102,7 @@
 	/**
 	 * Current request.
 	 *
-	 * @var HTTP\Request|HTTP\IncomingRequest|CLIRequest
+	 * @var HTTP\Request|HTTP\IncomingRequest|CLIRequest|RequestInterface
```

Comment:
I think this should all just be `ReqiestInterface`. @samsonasik ant compelling reason why the other hints would be needed?

Reply:
if method caller use CLIRequest or IncomingRequest only method, that needed, if not, it can only use  RequestInterface.